File: video/v1/room/create-group-room-with-h264/create-group-room-with-h264.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

room = @client.video.rooms.create(
                             status_callback: 'http://example.org',
                             type: 'group',
                             unique_name: 'DailyStandupWithH264Codec',
                             video_codecs: 'H264'
                           )

puts room.sid

File: video/v1/room/create-group-room-with-h264/create-group-room-with-h264.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Linq;
using Twilio;
using Twilio.Rest.Video.V1;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var videoCodecs = new [] {
            RoomResource.VideoCodecEnum.H264
        }.ToList();

        var room = RoomResource.Create(
            statusCallback: new Uri("http://example.org"),
            type: RoomResource.RoomTypeEnum.Group,
            uniqueName: "DailyStandupWithH264Codec",
            videoCodecs: videoCodecs
        );

        Console.WriteLine(room.Sid);
    }
}
